test/runnable/externmangle.d

```diff
@@ -169,6 +169,57 @@ interface Test38
      public static void dispose(ref Test38);
 }
 
+pragma(mangle, "class")
+extern(C++)
+struct S1
+{
+    private int val;
+    static S1* init(int);
+    int value();
+}
+
+pragma(mangle, "class")
+extern(C++)
+struct S2(T)
+{
+    private T val;
+    static S2!T* init(int);
+    T value();
+}
```

Comment:
Does `pragma(mangle, "class")` need to work for eponymous struct template? It's inconsistent with existing `pragma(mangle)` behavior for variables and functions.